scylla-server-rust/src/controller/driver_controller.rs

```diff
@@ -0,0 +1,37 @@
+use axum::{extract::State, Json};
+use serde::Serialize;
+
+use crate::{error::ScyllaError, prisma, services::driver_service, Database};
+
+use super::run_controller::{self, RunSend};
+
+#[derive(Serialize, PartialEq)]
+pub struct DriverSend {
+    username: String,
+    runs: Vec<run_controller::RunSend>,
```

Comment:
see this shouldnt be getting inmproted from the controller the object should exist outside of its respective controller.